bundle/phases/initialize.go

```diff
@@ -55,6 +55,7 @@ func Initialize() bundle.Mutator {
 				"workspace",
 				"variables",
 			),
+			permissions.PermissionDiagnostics(),
```

Comment:
If placement of this mutator after variable interpolation is intentional, please include a comment why.